CMakeLists.txt

```diff
@@ -45,7 +45,6 @@ list(APPEND CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/cmake/modules)
 
 # Dependencies
 include(cmake/Dependencies.cmake)
-include(cmake/ProtobufGenCpp.cmake)
```

Comment:
This overlaps with https://github.com/NVIDIA/DALI/pull/41

Reply:
Yes, because this PR includes (builds on) those two commits from #41.  Once that PR is merged I can rebase this onto master.

Reply:
Reasonable approach.